src/System.Drawing.Common/src/System/Drawing/Internal/GPStream.cs

```diff
@@ -107,7 +103,7 @@ public unsafe void Read(byte* pv, uint cb, uint* pcbRead)
 
             // Stream Span API isn't available in 2.0
 #if netcoreapp20
-            byte[] buffer = ArrayPool<byte>.Shared.Rent(Math.Max(DefaultBufferSize, checked((int)cb)));
+            byte[] buffer = ArrayPool<byte>.Shared.Rent(checked((int)cb));
```

Comment:
does this change have any perf implications? previously the number seemed to be power of 2 in many (most?) cases

Reply:
or perhaps we don't care - not sure in what scenarios we use this class